src/components/Web3Status/index.tsx

```diff
@@ -115,11 +114,6 @@ const Text = styled.p`
   font-weight: 500;
 `
 
-// we want the latest one to come first, so return negative if a is after b
-function newTransactionsFirst(a: TransactionDetails, b: TransactionDetails) {
-  return b.addedTime - a.addedTime
-}
-
```

Comment:
nit: this seems unrelated to base, it'd be better to have the linter fixes in their own PR instead of adding them to this one